.flake8

```diff
@@ -7,7 +7,7 @@ extend-ignore =
     # E203 whitespace before ':' (for compliance with black)
     E203,
 # for compliance with black
-max-line-length = 88
+max-line-length = 80
```

Comment:
Eish, I'm sorry, I just checked and it's actually 79... Maybe just remove this line instead, since we're using the `flake8` default